File: PYTHON/conditionProblems/con2.py
"""
  simple Hello world program
  By Itfyme.com
"""

def operation():
    print("enter score")
    score=int(input())
    if score<=56:
        print("failing")
    elif score<=65:
        print("unsatisfactory")
    else:
        print("satisfactory")
  
operation()

File: C/conditionProblems/con2.c
/* 
    multiply by 3.5
*/

// include library
#include <stdio.h>
int main(int argc, char *argv[]) {
  int score;
  printf("enter score\n");
  scanf("%d",&score);
  if(score<=56){
      printf("failing\n");
  }
  else if(score<=65){
      printf("unsatisfactory\n");
  }
  else{
      printf("satisfactory\n");
}
}
